projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/customer-ticketing-helpers/create-tickets.ts

```diff
@@ -0,0 +1,70 @@
+
+import * as commons from './customer-ticketing-commons';
+import {
+  TestTicketDetails,
+} from './customer-ticketing-commons';
+
+export function openCreateTicketPopup() {
+  cy.get('cx-customer-ticketing-list').should('exist');
+  cy.get('cx-customer-ticketing-create button').click();
+  cy.get('cx-customer-ticketing-create-dialog').contains('Add New Request');
+}
+
+
+export function fillTicketDetails(ticketDetails: TestTicketDetails){
+  const CATEGORY_SELECT = 0;
+
+  cy.get('cx-customer-ticketing-create-dialog').within(() => {
+    cy.get('.cx-customer-ticket-form-container').should('contain', 'Add New Request');
+    cy.get('textarea').first().type(ticketDetails.subject);
```

Comment:
I know this was already there, but I prefer not rely on the position of the input field.

```suggestion
    const SUBJECT_CONTROL = 'subject';
    cy.get(`textarea[formcontrolname="${SUBJECT_CONTROL}"]`).type(ticketDetails.subject);
```